packages/web-components/fast-foundation/docs/guide/aspnet.doc.md

```diff
@@ -0,0 +1,6 @@
+---
+id: aspnet
+title: ASP.NET
+sidebar_label: ASP.NET
+custom_edit_url: https://github.com/microsoft/fast-dna/edit/master/packages/web-components/fast-foundation/docs/guide/aspnet.doc.md
```

Comment:
Do we have content for these pages yet? If not, I would propose we add additional pages for `vue`, `react`, `angular`, and possibly `svelte`? It might be helpful to have even a bit of information and details on how to leverage custom elements with them.

Reply:
These are the frameworks/libraries that I personally know how to use and have existing, working setups for. The others we can add at any time if/when someone wants to put together a canonical example for that framework/library.